exercises/etl/test/test_etl.c

```diff
@@ -0,0 +1,121 @@
+#include "vendor/unity.h"
+#include "../src/etl.h"
+#include <stdlib.h>
+
+void setUp(void)
+{
+}
+
+void tearDown(void)
+{
+}
+
+static int compare_map(new_map * a, new_map * b, int len)
+{
+   if (a == NULL)
+      return (b == NULL) ? 0 : 1;
+   if (b == NULL)
+      return 1;
+   while (--len >= 0)
+      if ((a[len].key != b[len].key) || (a[len].value != b[len].value))
+         return 1;
+   return !(len == -1);
+}
+
+//Tests start here
```

Comment:
This comment is not needed. Comments that mark sections may seem like a good idea but are generally considered to be against the ideas of clean coding (see Uncle Bob, etc)